src/styles/aui/_aui.navigation-menu.scss

```diff
@@ -0,0 +1,427 @@
+    }
+
+
+    &.docked {
+      min-width: $menu-docked-width;
+      overflow: visible; // Since when docked the submenu's will appear to the right, show the overflowing content
+      width: $menu-docked-width;
+
+      .o-menu__tabs-wrapper {
+        box-shadow: none;
+      }
+
+      .o-menu-tab {
+        width: calc(#{$menu-docked-width} - 1px); // 1 px make up for border
+
+        .o-menu-tab__wrapper {
+          &:not(.active) {
+            overflow: hidden;
+          }
+        }
+      }
+
+      .o-menu-submenu {
+        left: $menu-docked-width;
+        position: absolute;
+        top: -1px; // Make up for border
+
+        .o-menu-submenu {
+          left: 100%;
+        }
+
+        .o-sub-menu__content-wrapper {
+          box-shadow: $box-shadow;
+          max-height: inherit;
+          transition: width ease-in-out;
+          width: 0;
+
+          .m-nav-list {
+            border: 1px solid $border-color;
+            border-bottom: none;
+            border-left: none;
+
+
+            .m-nav-list {
+              border-left: 1px solid $border-color;
+            }
+
+            li {
+
+              .a-icon {
+                transform: rotate(-90deg);
+              }
+
+              a {
+                border-bottom: 1px solid $border-color;
+                padding-left: $spacer-xs;
+              }
+            }
+          }
+        }
+
+        .o-sub-menu__content-wrapper {
+          width: $menu-undocked-width;
+        }
+      }
+    }
+  }
+}
+
+.o-menu .o-menu__link,
+.o-menu .m-nav-list a {
+  white-space: nowrap;
+
+  &,
+  &:active,
+  &:link,
+  &:hover {
+    color: $text-color-light;
+    text-decoration: none;
+  }
+}
+
+.o-menu > .o-menu__tabs-wrapper {
+  align-content: space-between;
+  align-items: center;
+  background: $white;
+  border-top: 1px solid $border-color;
+  display: flex;
+  position: relative;
+  width: 100%;
+  z-index: layer('header');
+
+  // Only display the first 2 items on a mobile screen
+  // and ignore the last element, since it will show a 'more...' menu tab
+  & > .o-menu-tab:nth-child(n + 3) {
+    &:not(:last-child) {
+      display: none; //We will need to  display the 'more...' tab
+
+      @include at($screen-sm) {
+        display: block; // Just display all on desktop since we will have enough space
+      }
+    }
+
+    &:last-child {
+      display: block;
+
+      @include at($screen-sm) {
+        display: none; // Don't display the 'more' tab on desktop
+      }
+    }
+  }
+
+  @include at($screen-sm) {
+    flex-direction: column;
+    height: auto;
+    width: auto;
+    z-index: layer('pane-overlay');
+  }
+}
+
+.o-menu .o-menu-tab {
+  border-right: 1px solid $border-color;
+  cursor: pointer;
+  flex-basis: 33%;
+  flex-grow: 1;
+  height: 100%;
+  position: relative;
+  width: 33%;
+
+  &:last-child {
+    border-right: none;
+  }
+
+  &.active {
+    .o-menu-tab__wrapper {
+      overflow-y: auto;
+    }
+
+    .a-tab-submenu-icon {
```

Comment:
`a-tab` bestaat niet apart, dus enkele voorstellen:
- `.o-menu__submenu-tab-icon`
- of gewoon via inheritance: `.fa`